### ISSUE TITLE
OnReplaced.java:41-49: Frontend and backend delimiters...

### ISSUE THREAD

**Issue by 0pdd** [objectionary/eo#2394]
The puzzle `2376-11b46788` from #2376 has to be resolved: 

https://github.com/objectionary/eo/blob/c4087fad04dd1d280415a09f034f839cd61c6059/eo-maven-plugin/src/main/java/org/eolang/maven/name/OnReplaced.java#L41-L49

The puzzle was created by rultor on 15-Aug-23. 

Estimate: 90 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by volodya-lombrozo** [comment]
@Graur Why is this a `Routine` task?  It influence the solution with versions significantly.

**Comment by volodya-lombrozo** [comment]
@Graur Yes, you are right. It is mostly a refactoring task, but it also involves an architectural [violation](https://github.com/objectionary/eo/issues/2382#issuecomment-1674769152). I would recommend solving it as soon as possible while the cost is lower. In a few months, this task will cost significantly more. However, I don't insist here.

**Comment by Graur** [comment]
@volodya-lombrozo Ok, let's make it happened

**Comment by 0pdd** [comment]
@0pdd the puzzle [#2420](https://github.com/objectionary/eo/issues/2420) is still not solved.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#2463](https://github.com/objectionary/eo/issues/2463) is still not solved; solved: [#2420](https://github.com/objectionary/eo/issues/2420).

**Comment by 0pdd** [comment]
@0pdd all 3 puzzles are solved here: [#2420](https://github.com/objectionary/eo/issues/2420), [#2463](https://github.com/objectionary/eo/issues/2463), [#2544](https://github.com/objectionary/eo/issues/2544).
